src/logic/safe/store/selectors/pendingTransactions.ts

```diff
@@ -8,10 +8,10 @@ import {
   LocalTransactionStatus,
   Transaction,
 } from 'src/logic/safe/store/models/types/gateway.d'
-import { LocalStatusesState, LOCAL_TRANSACTIONS_ID } from '../reducer/localTransactions'
+import { PendingTransactionsState, PENDING_TRANSACTIONS_ID } from 'src/logic/safe/store/reducer/pendingTransactions'
 
-export const localStatuses = (state: AppReduxState): LocalStatusesState => {
-  return state[LOCAL_TRANSACTIONS_ID]
+export const localStatuses = (state: AppReduxState): PendingTransactionsState => {
+  return state[PENDING_TRANSACTIONS_ID]
```

Comment:
Is this state already filtered for the current chain id?

Reply:
No

Reply:
Now that I've converted the pending transaction state to `Set<safeTxHash>`, it needn't matter.

Reply:
True. I'll adjust it. To answer the previous question: it's the entire state.

Reply:
This still stand?

Reply:
This function returns the whole pending state. It's only used to select it for persisting the status.

There is a selector to filter by the current `chainId` below, `pendingTxs` but it is only used to abstract logic in other selectors so it is not exported.

Are they not aptly named?

Reply:
I see now. The selector below (on line 18) is what I though this was.

I would swap the names, yes.